projects/xpdf/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER david@adalogics.com
+RUN apt-get update && apt-get install -y make wget cmake libqt4-dev
+RUN wget https://xpdfreader-dl.s3.amazonaws.com/xpdf-4.02.tar.gz
```

Comment:
Can you ask xdf authors if there is a link to trunk brnach, otherwise this would require continuous changes to this file. fine for now, please fix in another PR.